Formula/vulkan-headers.rb

```diff
@@ -4,6 +4,7 @@ class VulkanHeaders < Formula
   url "https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.248.tar.gz"
   sha256 "e2a3ef209deee3e47e67f3e57846f343c53b96fbe854f5c7427f0bfee478907f"
```

Comment:
If the tag was updated in place, then shouldn't the checksum change?

Reply:
That is a good point, and I'll correct that!  I'm not sure what I was thinking.